Application/EdFi.Ods.AdminApi/Infrastructure/Database/Queries/GetApplicationsQuery.cs

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Admin.DataAccess.Contexts;
+using EdFi.Admin.DataAccess.Models;
+using EdFi.Ods.AdminApi.Infrastructure.Extensions;
+using Microsoft.EntityFrameworkCore;
+
+namespace EdFi.Ods.AdminApi.Infrastructure.Database.Queries
```

Comment:
Use the namespace without braces, sames as Application/EdFi.Ods.AdminApi/Infrastructure/Database/Queries/GetVendorsQuery.cs

Example:
`namespace EdFi.Ods.AdminApi.Infrastructure.Database.Queries;`